### PR TITLE
adds Dylan Mc article

### PR DESCRIPTION
**Pull request by dylnmc**
Hi romainl,

Here is an article. If you like it, then I hope it's put to good use. Otherwise, let me know if I should make any changes (it could still probably use some work, but it's what I spit out).

Cheers,
dylnmc

### CODE REVIEW

File: dylnmc-operator-pending.md [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+text object. In vimtutor, this concept is introduced pretty early on. Text
+objects are "commands that can only be used while in Visual mode or after an
+operator", as is explained in [:help text-objects][help-text-objects].
+Essentially, text objects are descriptors that tell an action what to operate
+on. They do this by creating a visual selection that the action then operates
+on.
+
+One of the most commonly-used examples is `iw`, which is short for or
+**inner-word**. If you type `diw`, for instance, this will delete the word under
+the cursor. `d` is the action: **delete**; `iw` is the text object:
```
Comment (romainl) [inline dylnmc-operator-pending.md]: "action" should be "operator"

File: dylnmc-operator-pending.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+Of course, the *LHS* is `il`, which is what we must press in visual-mode or when
+there is a pending operator in order to execute the *RHS* and fulfil our text
+object's purpose. This can be anything we want, but I find that `il` for
+**inner-line** makes the most sense. You can even use special-keys, such as
+`<F12>` or `<c-bslash>`, to map special keys. However, note that there are very
+limited options and vim maps quite a few of them already in visual mode.
+
+#### *inner-line* RHS: `:<c-u>normal! g_v^<cr>`
+
+The `:` begins command mode, and it operates as if you, the user, had typed `:`
```
Comment (romainl) [inline dylnmc-operator-pending.md]: It's "command-line mode". See `:help vim-modes`.

**Review by romainl — COMMENTED** [review]
Hi, Dylan. This is a very very nice contribution!

**Comment by dylnmc** [comment]
@romainl updated.

---

If I should create a pull request [here](https://github.com/vim-advent-calendar/vimways.org/pulls), let me know.

**Comment by romainl** [comment]
@dylnmc yes, please do. Your article is very good.

**Pull request closed without merging**